mapset/index.php

```diff
@@ -1,6 +1,14 @@
 <?php
 	$mapset_id = $_GET['mapset_id'] ?? -1;
 	require '../base.php';
+
+	$foundSet = false;
+	$result = $conn->query("SELECT * FROM `beatmaps` WHERE `SetID`='${mapset_id}' AND `mode`='0' ORDER BY `SR` DESC;");
+	$sampleRow = $result->fetch_assoc();
+	mysqli_data_seek($result, 0);
+
+	$PageTitle = htmlspecialchars($sampleRow['Title']) . " mapped by " . GetUserNameFromId($sampleRow['CreatorID'], $conn);
```

Comment:
```suggestion
	$PageTitle = htmlspecialchars($sampleRow['Title']) . " by " . GetUserNameFromId($sampleRow['CreatorID'], $conn);
```

just to match the page's `<h1>` tag

Reply:
yeah